topology/University of Wisconsin/CHTC/CHTC.yaml

```diff
@@ -762,4 +762,39 @@ Resources:
         Details:
           hidden: false
 
+  CHTC-submit6:
+    Active: true
+    ContactLists:
+      Administrative Contact:
+        Primary:
+          ID: 46a55ac4815b2b8c00ff283549f413113b45d628
+          Name: Aaron Moate
+        Secondary:
+          ID: 3f306d87236d84ef770ddf0c34844908e2d94dfa
+          Name: Timothy Slauson
+      Resource Report Contact:
+        Primary:
+          ID: 46a55ac4815b2b8c00ff283549f413113b45d628
+          Name: Aaron Moate
+        Secondary:
+          ID: 3f306d87236d84ef770ddf0c34844908e2d94dfa
+          Name: Timothy Slauson
+      Security Contact:
+        Primary:
+          ID: 46a55ac4815b2b8c00ff283549f413113b45d628
+          Name: Aaron Moate
+        Secondary:
+          ID: 3f306d87236d84ef770ddf0c34844908e2d94dfa
+          Name: Timothy Slauson
+    Description: Login node for OSPool, see OPS-198
+    FQDN: submit6.chtc.wisc.edu
+    ID: 1191
+    Services:
+      Submit Node:
+        Description: OSPool submit node, see OPS-198
+        Details:
+          hidden: false
+    VOOwnership:
+      GLOW: 100
```

Comment:
```suggestion
```

Aren't we submitting OSG VO jobs from here? I'd argue that we should drop this, then